### DISCUSSION TITLE
Fan Speed of 1 equals full fan speed

### DISCUSSION THREAD

**Discussion by quizilkend** [vmatare/thinkfan#159] [Support]
Hi,
I succeeded in installing and configuring thinkfan.
Unfortunately it seems that no matter what fan-level (tpacpi-fan) I set, the fan always spins at full speed.
I run thinkfan from the debian buster repository in version 1.2.1 on an X1 Carbon 4th Gen (intel 6600u cpu).

Do you have an idea, what the problem could be?

Best regards
Samuel

**Comment by vmatare** [comment]
Sorry, my crystal ball is out for repairs. Honestly though, I'll need to see your config and the output of `journalctl -el -b0 -u thinkfan` (if you run thinkfan as a service) or the terminal output if you start it with `-n`.
